### PR TITLE
Add XML Converter to Market Roles

### PR DESCRIPTION
**Pull request by thbaan**
## References

* #0000

### CODE REVIEW

**Review by Korgath — APPROVED** [review]

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDIMessaging/XmlConverter/Mappings/ChangeOfSupplierXmlMappingConfiguration.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+{
+    public class ChangeOfSupplierXmlMappingConfiguration : XmlMappingConfigurationBase
+    {
+        public ChangeOfSupplierXmlMappingConfiguration()
+        {
+            CreateMapping<RequestChangeOfSupplier>("MktActivityRecord", mapper => mapper
+                .AddProperty(x => x.MeteringPointId, "marketEvaluationPoint.mRID")
+                .AddProperty(x => x.StartDate, "start_DateAndOrTime.date")
+                .AddProperty(x => x.EnergySupplierId, "marketEvaluationPoint.energySupplier_MarketParticipant.mRID")
+                .AddProperty(x => x.ConsumerId, "marketEvaluationPoint.customer_MarketParticipant.mRID")
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDIMessaging/XmlConverter/Mappings/ChangeOfSupplierXmlMappingConfiguration.cs:27]: This also needs to account for VAT/CPR, as you do in the requestmovein config

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDIMessaging/XmlConverter/Mappings/ChangeOfSupplierXmlMappingConfiguration.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+using Energinet.DataHub.MarketRoles.Application.ChangeOfSupplier;
+
+namespace Energinet.DataHub.MarketRoles.Infrastructure.EDIMessaging.XmlConverter.Mappings
+{
+    public class ChangeOfSupplierXmlMappingConfiguration : XmlMappingConfigurationBase
+    {
+        public ChangeOfSupplierXmlMappingConfiguration()
+        {
+            CreateMapping<RequestChangeOfSupplier>("MktActivityRecord", mapper => mapper
+                .AddProperty(x => x.MeteringPointId, "marketEvaluationPoint.mRID")
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDIMessaging/XmlConverter/Mappings/ChangeOfSupplierXmlMappingConfiguration.cs:24]: Shouldn't this also be accountpoint gsrn, just as in the move in config?

**Review by erduna — CHANGES_REQUESTED** [review]
for change of supplier we also need to consider VAT/SSN

**Review by cmgrd — APPROVED** [review]

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDIMessaging/XmlConverter/Mappings/ChangeOfSupplierXmlMappingConfiguration.cs (hunk 1 above)
Comment (thbaan) [reply source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDIMessaging/XmlConverter/Mappings/ChangeOfSupplierXmlMappingConfiguration.cs:27]: This is out of scope of this PR. We should take care of this in a seperate task/story.

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDIMessaging/XmlConverter/Mappings/ChangeOfSupplierXmlMappingConfiguration.cs (hunk 2 above)
Comment (thbaan) [reply source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDIMessaging/XmlConverter/Mappings/ChangeOfSupplierXmlMappingConfiguration.cs:24]: This is out of scope of this PR. We should take care of this in a seperate task/story.

**Pull request merged**